lib/Myriad/API.pm

```diff
@@ -60,6 +63,21 @@ method service_by_name ($name) {
     );
 }
 
+
+=head2 config
+
+Returns a L<Ryu::Observable> that hold the value of the configuration.
+
+=cut
+
+method config ($key) {
+    my $pkg = caller;
+    if($Myriad::Config::SERVICES_CONFIG{$pkg}->{$key}) {
```

Comment:
This is wrong - config is per-instance

Reply:
yes and that is handled in `Myriad::Config` this is not to look up for the config value this is just to check if the developer declared this before or not.